### ISSUE TITLE
issue about file bbox.c

### ISSUE THREAD

**Issue by JackShi001** [PKU-ICST-MIPL/Saliency-guided-Faster-R-CNN#1]
Hi, I tried to run your code, but it returns error when bbox was being involved. I found out that the bbox file is in C programming. Do I need to compile the c file first? And besides, how to use the three setup files, is there any specific answer to use them?

Really looking forward to your response. Thanks!

**Comment by wzhiyan** [comment]
I can't run this code either，have you solved it ？

**Comment by JackShi001** [comment]
Nope, I haven't figure it out. I have sent message to the author but have not received any response yet.

**Comment by doublemanyu** [comment]
I meet the same problem now. I try to fix it by this way. I run Makefile in Ubuntu14.0 use the command"make"，and I  use bbox.pyx and cpu_nms.pyx  into ./rcnn/cython/,
